languages.toml

```diff
@@ -2421,3 +2421,16 @@ language-server = { command = "nimlangserver" }
 [[grammar]]
 name = "nim"
 source = { git = "https://github.com/aMOPel/tree-sitter-nim", rev = "240239b232550e431d67de250d1b5856209e7f06" }
+
+[[grammar]]
+name = "just"
+source = { git = "https://github.com/IndianBoy42/tree-sitter-just", rev = "8af0aab79854aaf25b620a52c39485849922f766" }
+
+[[language]]
+name = "just"
+scope = "source.make"
```

Comment:
Is this scope still the correct one ?

Reply:
Thanks, classic copy paste I guess :blush: